mitodl/odl-video-service
# Simplified video serializer for collection page

#### What are the relevant tickets?
Partially fixes #512 (paging will do the rest)

#### What's this PR do?
Uses a simplified video serializer on the collection detail page with just the fields required for the various dialogs.

#### How should this be manually tested?
Go to a collection page with lots of videos.  The page should load normally and all the video dialogs on the cards should still work ('Edit', 'Share', etc).  Note the size of the API response that gets the collection's video details in the network tab.  Try again on the master branch and compare the API response sizes, it should be substantially smaller on this PR branch.
